packages/main/src/plugin/kubernetes/context-health-checker.ts

```diff
@@ -111,4 +111,8 @@ export class ContextHealthChecker implements Disposable {
   public getState(): ContextHealthState {
     return this.#currentState;
   }
+
+  private isAbortError(err: unknown): boolean {
```

Comment:
```suggestion
  private isAbortError(err: unknown): err is AbortError {
```

Reply:
But if we do this, we need again the AbortError type. I wanted to get rid of it

Reply:
:/